shell/app/styles/antd-extension.scss

```diff
@@ -407,6 +407,14 @@ div.ant-tabs-nav .ant-tabs-tab {
   margin-right: 40px;
 }
 
+div.ant-tabs-nav::before, .ant-tabs-nav::after {
+  display: block;
```

Comment:
What's this for?

Reply:
antd3.x styles makes display table, causing the the border not to work.